safe_eth/safe/tests/api/test_transaction_service_api.py

```diff
@@ -36,11 +49,37 @@ def test_constructor(self):
         ethereum_network = EthereumNetwork.SEPOLIA
         base_url = "https://safe.global"
```

Comment:
Because we have a previous API url without authentication, could you try to configure any network and do a get to any endpoint?